core/src/main/java/nu/yona/server/CoreConfiguration.java

```diff
@@ -75,7 +75,8 @@ RepositoryProvider repositoryProvider()
 	public ReloadableResourceBundleMessageSource messageSource()
 	{
 		ReloadableResourceBundleMessageSource messageBundle = new ReloadableResourceBundleMessageSource();
-
+		
+		messageBundle.setFallbackToSystemLocale(false);
```

Comment:
OK, understood. Let's leave it in. I'll try to remember to mention it in the comment when merging the PR.